strings/turkish.txt

```diff
@@ -52,15 +52,15 @@ Görüntü Ayarları
 Ses Ayarları
 Kontrol Ayarları
 Yükle/Kayıt Rekor
-Krediye Bak
+Yapımcıları Göster
 Araba Sayısı
 Rastgele Araba
 Rastgele Pist
 Nesne Alma
 Tur Sayısı
 Rakip Etiketleme Zamanı
 Zorluk
-Açılan Araba 
+Kilidi Açılan Araba 

```

Comment:
This line for "Access Cars" is not used anywhere in the game, and its original intended purpose is unknown. I'm keeping the previous version for now.